test/NuGet.Clients.Tests/NuGet.MSSigning.Extensions.Test/NuGetMSSignCommandTest.cs

```diff
@@ -255,7 +289,7 @@ public void MSSignCommandArgParsing_ValidTimestampHashAlgorithm(string timestamp
                 var packagePath = Path.Combine(dir, "package.nupkg");
                 var timestamper = "https://timestamper.test";
                 var certFile = Path.Combine(dir, "cert.p7b");
-                var certificateFingerprint = new Guid().ToString();
+                var certificateFingerprint = _sha1Hash;
```

Comment:
When testing one validation --- in this case, timestamp algorithm --- amongst many, it's important to pass reasonably valid values for all other (non-tested) validations to isolate the validation under test.

Previously, we passed a GUID string for certificate fingerprint, but now we pass a SHA-1 fingerprint.  This PR changes fingerprint validation, specifically for SHA-1 fingerprints, so it's doubly weird that we're passing a deprecated value for a property we're not even testing.

I recommend creating a fake SHA-256 fingerprint and using it wherever we're _not_ testing certificate fingerprint validation.

Reply:
Thanks for the feedback. Updated tests